t/page/missing_country.rb

```diff
@@ -0,0 +1,15 @@
+require 'minitest/autorun'
+require_relative '../../lib/page/missing_country'
+require 'pry'
+
+describe 'MissingCountry' do
+  it 'a missing country has a display name' do
+    page = Page::MissingCountry.new('eritrea')
+    assert_equal page.country[:displayName], 'Eritrea'
```

Comment:
Again, I'd prefer we keep our syntax consistent across all the test files, rather than switching between `assert_equal` and `.must_equal`. (Open to persuasion that it should be `assert` style by default, but until then better to stay the same everywhere)